hinemosu1415/server/server.js

```diff
@@ -3,96 +3,82 @@ import { Hono } from 'hono'
 import { cors } from 'hono/cors'
 import { vValidator } from '@hono/valibot-validator'
 import { object, string, boolean } from 'valibot'
-import Database from 'better-sqlite3'
+import pkg from 'pg'
 
-const app = new Hono()
+const { Pool } = pkg
 
+const app = new Hono()
 app.use('*', cors())
 
-const db = new Database('db/todos.db')
-
-db.prepare(`
-  CREATE TABLE IF NOT EXISTS todos (
-    id INTEGER PRIMARY KEY AUTOINCREMENT,
-    title TEXT NOT NULL,
-    completed INTEGER NOT NULL
-  )
-`).run()
-
+const pool = new Pool({
+  user: process.env.PGUSER,
+  host: process.env.PGHOST,
+  database: process.env.PGDATABASE,
+  password: process.env.PGPASSWORD,
+  port: Number(process.env.PGPORT)
+})
 
 const TodoInputSchema = object({
   title: string(),
   completed: boolean()
 })
 
-app.post('/todo', vValidator('json', TodoInputSchema), async (c) => {
-  const data = c.req.valid('json')
-
-  try {
-    const stmt = db.prepare('INSERT INTO todos (title, completed) VALUES (?, ?)')
-    const info = stmt.run(data.title, data.completed ? 1 : 0)
-
-    return c.json({ success: true, id: info.lastInsertRowid })
-  } catch (err) {
-    console.error('Todoの保存中にエラーが発生しました:', err)
-    return c.json({ success: false, error: 'サーバー内部でエラーが発生しました' }, 500)
-  }
+app.get('/todo', async (c) => {
+  const result = await pool.query('SELECT * FROM todos')
```

Comment:
```suggestion
  const conn = await pool.connect()
  const result = await conn.query('SELECT * FROM todos')
```

`pool` をそのまま使うのではなくて毎回コネクションを取ろう